src/openrct2/interface/Widget.h

```diff
@@ -60,6 +61,57 @@ enum
 
 constexpr uint8_t SCROLLBAR_WIDTH = 10;
 
+constexpr rct_widget MakeWidget(
+    ScreenCoordsXY origin, ScreenSize size, uint8_t type, uint8_t colour, uint32_t content = 0xFFFFFFFF,
+    rct_string_id tooltip = STR_NONE)
+{
+    rct_widget out = {};
+    out.left = origin.x;
+    out.right = origin.x + size.width - 1;
+    out.top = origin.y;
+    out.bottom = origin.y + size.height - 1;
+    out.type = type;
+    out.colour = colour;
+    out.content = content;
+    out.tooltip = tooltip;
+
+    return out;
+}
+
+constexpr rct_widget MakeRemapWidget(
+    ScreenCoordsXY origin, ScreenSize size, uint8_t type, uint8_t colour, uint32_t content = 0xFFFFFFFF,
+    rct_string_id tooltip = STR_NONE)
+{
+    return MakeWidget(origin, size, type, colour, IMAGE_TYPE_REMAP | content, tooltip);
+}
+
+#define MakeStepperWidgets(...)                                                                                                \
+    MakeWidget(__VA_ARGS__), MakeStepperDecreaseWidget(__VA_ARGS__), MakeStepperIncreaseWidget(__VA_ARGS__)
+
+constexpr rct_widget MakeStepperDecreaseWidget(
+    ScreenCoordsXY origin, ScreenSize size, [[maybe_unused]] uint8_t type, uint8_t colour,
+    [[maybe_unused]] uint32_t content = 0xFFFFFFFF, rct_string_id tooltip = STR_NONE)
+{
+    const int16_t xPos = origin.x + size.width - 25;
+    const int16_t yPos = origin.y + 1;
+    const uint16_t width = 12;
+    const uint16_t height = size.height - 2;
+
+    return MakeWidget({ xPos, yPos }, { width, height }, WWT_BUTTON, colour, STR_NUMERIC_DOWN, tooltip);
+}
+
+constexpr rct_widget MakeStepperIncreaseWidget(
```

Comment:
These are called "spinner" in our code. Why are using a different term here? That's only going to cause confusion - the code should use one term.

(If you believe the spinner name is incorrect/worse than stepper, it should be an issue of its own, and it should be renamed throughout.)

Reply:
Thanks. No, I did not intend to change the naming. Now that you mention it, I remember the discussion from #7579 and did not mean to drag that up again. I will change it to match the old naming tonight.